pitch.md

```diff
@@ -1,44 +1,90 @@
-# Project Title (replace me)
+# BarScript
 
 ## Overview
 _What's this project about, briefly?_
 
+This project is about creating a language to specify coffee/alcoholic drinks, so they can be made to your taste by an automated machine. We already have some automated coffee machines, but these machines usually have a limited number of adjustable settings and are considered by many to be inferior to an actual barista's work for this reason. Theoretically, if a project like this was wildly successful it'd eliminate the need for bartenders/baristas.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Coffee drinks are difficult to make. In the world of the ever-present hipster, it's getting harder and harder to get it just right. In a world where customers actually order a "split-shot, heavy on decaf, 20 oz, soy, extra dry latte, with half pump vanilla & half-pump hazlenut", keeping track of all the modifications to make for a coffee drink is a difficult experience even for a physical barista. On the other hand, automatic coffee machines have a finite amount of settings that a customer can customize. 
+
+Alcoholic beverages present a similar challenge. The barista I was discussing this project idea with told me about his favourite drink: "Old fashioned, with 2 shots of rye whisky, cane syrup instead of brown sugar, extra bitters, and orange zest instead of an orange slice". 
+
+If there was a standard language by which a customer could specify a drink to an arbitrary level of precision, then you could easily program an automatic coffee machine or autobartender to make exactly the drink you want. 
```

Comment:
I think users will be giving the script to the coffee machine, at least for this version of the project.